### ISSUE TITLE
Cheat Engine 7.2 doesn't open.

### ISSUE THREAD

**Issue by TallisManeiro** [cheat-engine/cheat-engine#1660]
I already do a driverquery, here: https://pastebin.com/qX2d0gmZ

When i try to execute de installer launcher it just disappear, idk what to do, can anyone help me?

**Comment by debug-jay** [comment]
Same here but windows 10, i dont have Badlion, anti virus off etc

**Comment by TallisManeiro** [comment]
Yeah, i don't even installed badlion. Did you actually know what to do?

**Comment by TallisManeiro** [comment]
> I already do a driverquery, here: https://pastebin.com/qX2d0gmZ
> 
> When i try to execute de installer launcher it just disappear, idk what to do, can anyone help me?
> 
> edit - all my antivirus are disabled, i use an windows 7 system, idk if this matters

HEY BROS!!! TO PEOPLE THAT CAN'T INSTALL CHEATENGINE I FOUND A METOD. Install this: https://mega.nz/file/28ZhWYzL#t3lm245aBCUJ5jnRF1YXxfEjrtssHAtN2bPh2Wo0rnc and after install just execute the .exe nammed "cheat engine", and done! You can now use cheat engine.
